README.md

```diff
@@ -0,0 +1,58 @@
+# Backend
+
+## Using the backend for the first time
+
+After cloning the backend to your local repository you need to perform the following steps, in order to start start the flask application.
+
+<ol>
+    <li>
+        Create a python virtual environment (venv) in your backend directory:
+        <ul>
+            <li>
+                (terminal): python3 -m venv .venv or python -m venv .venv
+            </li>
+        </ul>
+    </li>
+    <li>
+        Install all necessary dependencies with the provided requirements.txt file:
+        <ol>
+            <li>
+                Change into the newly created virtual environment:
+                <ul>
+                    <li>
+                        (terminal): cd .venv
+                    </li>
+                </ul>
+            </li>
+            <li>
+                Activate the enviroment
+                <ul>
+                    <li>
+                        (terminal): source bin/activate<br>
+                        or (windows cmd): .\Scripts\Activate.ps1
+                    </li>
+                </ul>
+            </li>
+            <li>
+                Install the required dependencies:
+                <ul>
+                    <li>
+                        (terminal): pip install -r ../requirements.txt
+                    </li>
+                </ul>
+            </li>
+        </ol>
+    </li>
+    <li>
+        To start the application:
```

Comment:
To start the app, one needs to be in the `src` directory. If one ist still in the directory from the previous step, it won´t work. Maybe clarify, that one must switch to `src`.

Reply:
I placed a hint